airgun/browser.py

```diff
@@ -136,7 +136,7 @@ def post_init(self):
             )
         # Workaround maximize_window() not working with chrome in docker
         if not (self.provider == 'docker' and
-                self.browser == 'chrome'):
+                self.browser == 'chrome' or self.provider == 'selenium'):
```

Comment:
We're testing if running some tests in selenium grid wouldn't be faster than in saucelabs so I needed web driver not to maximize a window, there are no active browser window available.

Reply:
if you are just testing something, so do it in your branch, not in the master

Reply:
Feel free to test whatever you wish locally, but never submit PRs to master branch which will affect others just for testing purposes :)
`if provider is not 'selenium' then maximize a window` - means it will affect local runs for everyone, and, most likely, will cause failures since small window will cause elements shift & some of them will appear out of the screen.

Reply:
There is some misunderstanding here. This was already tested locally. This PR is to add functionality to AirGun so anybody can use it this way.

And regarding this specific change: our selenium grid setup uses selenium workers in docker containers, so although `self.browser` is 'selenium', we need to workaround same problem as the original `if` is workarounding.

Reply:
Yeah, there's definitely misunderstanding here. I trust you it was tested in one scenario, but not in others, as it will add regression for sure. Once again:
`self.provider`, according to docstring, can be 1 of `('selenium', 'docker', 'saucelabs')`.
Now to the logic itself:
```python
if not (condition_1 or self.provider == 'selenium'):
    maximize_window()
```
So, if provider is not 'selenium' (so it's 'docker' or 'saucelabs') - then maximize the window. Therefore, such condition introduces regression - for local runs with no saucelabs/docker you specify 'selenium', and with this change it means window won't get maximized anymore.

Reply:
Not to mention general idea is wrong. Isn't it possible to maximize window in remote webdriver? It definitely is. In Silvia's case it's docker issue, not remote webdriver one, so prohibitin maximizing window for all remote webdriver runs is just wrong.

Reply:
I see, got you @abalakh ! @schlupov, it could be something like:

-if not (self.provider == 'docker' and self.browser == 'chrome'):
+if not (self.provider in ('docker', 'selenium') and self.browser == 'chrome'):

Ad for general idea: would it be better to come with 'grid_docker' provider? This comes from overloading the 'selenium' provider in this case.